exchanges/bitstamp/bitstamp_websocket.go

```diff
@@ -305,14 +285,18 @@ func (b *Bitstamp) Subscribe(channelsToSubscribe subscription.List) error {
 	return errs
 }
 
-// Unsubscribe sends a websocket message to stop receiving data from the channel
-func (b *Bitstamp) Unsubscribe(channelsToUnsubscribe subscription.List) error {
+// Unsubscribe sends a websocket message to stop receiving data from a list of channels
+func (b *Bitstamp) Unsubscribe(subs subscription.List) error {
+	return b.ParallelChanOp(subs, b.unsubscribe, 1)
+}
+
+func (b *Bitstamp) unsubscribe(subs subscription.List) error {
 	var errs error
-	for _, s := range channelsToUnsubscribe {
+	for _, s := range subs {
 		req := websocketEventRequest{
 			Event: "bts:unsubscribe",
 			Data: websocketData{
-				Channel: s.Channel,
+				Channel: s.QualifiedChannel,
```

Comment:
I am surprised this works with auth subs given that to sub you add an ID. Could you please add a case to wsHandleData though?
```
[DEBUG] | WEBSOCKET | 05/12/2024 10:41:05 | Bitstamp wss://ws.bitstamp.net: Message received: {"event":"bts:unsubscription_succeeded","channel":"my_trades_xrpusd","data":{}}
[DEBUG] | WEBSOCKET | 05/12/2024 10:41:05 | Bitstamp wss://ws.bitstamp.net: Message received: {"event":"bts:unsubscription_succeeded","channel":"my_trades_btceur","data":{}}
[DEBUG] | WEBSOCKET | 05/12/2024 10:41:05 | Bitstamp wss://ws.bitstamp.net: Message received: {"event":"bts:unsubscription_succeeded","channel":"my_trades_btcusd","data":{}}
[DEBUG] | WEBSOCKET | 05/12/2024 10:41:05 | Bitstamp wss://ws.bitstamp.net: Message received: {"event":"bts:unsubscription_succeeded","channel":"my_trades_eurusd","data":{}}
[DEBUG] | WEBSOCKET | 05/12/2024 10:41:05 | Bitstamp wss://ws.bitstamp.net: Message received: {"event":"bts:unsubscription_succeeded","channel":"my_trades_xrpeur","data":{}}
[WARN]  | WEBSOCKET | 05/12/2024 10:41:05 | Bitstamp - Unhandled websocket message: {"event":"bts:unsubscription_succeeded","channel":"my_trades_xrpusd","data":{}}
[WARN]  | WEBSOCKET | 05/12/2024 10:41:05 | Bitstamp - Unhandled websocket message: {"event":"bts:unsubscription_succeeded","channel":"my_trades_btceur","data":{}}
[WARN]  | WEBSOCKET | 05/12/2024 10:41:05 | Bitstamp - Unhandled websocket message: {"event":"bts:unsubscription_succeeded","channel":"my_trades_btcusd","data":{}}
[WARN]  | WEBSOCKET | 05/12/2024 10:41:05 | Bitstamp - Unhandled websocket message: {"event":"bts:unsubscription_succeeded","channel":"my_trades_eurusd","data":{}}
[WARN]  | WEBSOCKET | 05/12/2024 10:41:05 | Bitstamp - Unhandled websocket message: {"event":"bts:unsubscription_succeeded","channel":"my_trades_xrpeur","data":{}}
```

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/033524ae3

I seem to recall I was surprised as well, but it seemed to work.
When Bitstamp is working again I'll exercise it with some private trades to make sure the unsubscribe is real.

Reply:
I've manually confirmed this is not unsubscribing:
```
< {"event":"bts:subscription_succeeded","channel":"private-my_trades_btcusd-3141592","data":{}}
> {"data":{...,"order_type":1,"order_subtype":5,"datetime":"1733705910","microtimestamp":"1733705909775000","amount":0.00019992,"amount_str":"0.00019992","amount_traded":"0","amount_at_create":"0.00019992","price":102000,"price_str":"102000",....3},"channel":"private-my_orders_btcusd-3141592","event":"order_created"}
< {"event":"bts:unsubscription_succeeded","channel":"my_trades_btcusd","data":{}}
```
5 seconds later:
```
>  {"data":{..."order_type":1,"order_subtype":5,"datetime":"1733705939","microtimestamp":"173370
5938584000","amount":0.00019992,"amount_str":"0.00019992","amount_traded":"0","amount_at_create":"0.00019992","price":102000,"price_str":"102000","trade_account_id":....},"channel":"priv
ate-my_orders_btcusd-3141592","event":"order_deleted"}
```

Reply:
I'm okay that it's not a supported feature. I just didn't want the `warn` logs.
Also the sub you say is:
```
my_trades_btcusd
```
The channel that data is from is:
```
priv
private-my_orders_btcusd
```

Reply:
> Also the sub you unsubscribe from is:
I copied the wrong line out of many :)

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/b44055fbc

Reply:
Note: I haven't introduced a MockWS test for auth subs, but I've tested it manually to ensure I see my own trades streamed whilst subscribed, and then after unsub they stop streaming.